### PR TITLE
refactor(launch): restrict chain name to only lowercase character

### PR DESCRIPTION
**Pull request by lubtd**
Restrict the chain name to only lowercase character

### CODE REVIEW

File: x/launch/types/chain.go [hunk 1]
```diff
@@ -9,7 +9,10 @@ import (
 	codec "github.com/cosmos/cosmos-sdk/codec/types"
 )
 
-const chainIDSeparator = "-"
+const (
+	ChainIDSeparator   = "-"
+	ChainNameMaxLength = 30
```
Comment (Pantani) [inline x/launch/types/chain.go:14]: Is this const need to be public/exported?

**Review by Pantani — APPROVED** [review]
💪🏽

only one doubt

File: x/launch/types/chain.go (hunk 1 above)
Comment (lubtd) [reply x/launch/types/chain.go:14]: In order to use it in tests for edge case (trying to create a name with `ChainNameMaxLength +1`)

**Review by ilgooz — APPROVED** [review]
